Libraries/LinkingIOS/RCTLinkingManager.h

```diff
@@ -11,17 +11,17 @@
 
 @interface RCTLinkingManager : RCTEventEmitter
 
-+ (BOOL)application:(UIApplication *)app
-            openURL:(NSURL *)URL
-            options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options;
++ (BOOL)application:(nonnull UIApplication *)app
+            openURL:(nonnull NSURL *)URL
+            options:(nullable NSDictionary<UIApplicationOpenURLOptionsKey, id> *)options;
```

Comment:
Looks like this parameteris `nonnull`.

https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623112-application